latest/glibc/Dockerfile.builder

```diff
@@ -49,14 +49,16 @@ RUN set -eux; \
 
 WORKDIR /usr/src/busybox
 
-# https://github.com/docker-library/busybox/issues/198
-# https://bugs.busybox.net/show_bug.cgi?id=15931
-# https://bugs.debian.org/1071648
+# apply necessary/minimal patches (see /.patches/ in the top level of the repository)
+COPY \
+	/.patches/no-cbq.patch \
+	/.patches/sha1_process_block64_shaNI.patch \
```

Comment:
The explicit `/` prefix here is not my typical style, but was an intentional choice to try and make it more clear that these come from the _root_ of the repository, not the directory of `Dockerfile.builder` -- thoughts on that justification are welcome!

Reply:
I don't have a better alternative, and it seems reasonable, so LGTM.